dot-object/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Dot-Object v1.4.2
+// Type definitions for Dot-Object v1.4.3
```

Comment:
We only use MAJOR.MINOR for version numbers. Can you strip the last portion off?